File: ios/Classes/DebugPrint/DebugPrint.m
//
//  DebugPrint.m
//  qweather
//
//  Created by CyJay on 2021/4/19.
//


#import "DebugPrint.h"

@implementation DebugPrint
static BOOL _debug = NO;
+ (void)setDebug:(BOOL)debug{
    if (debug)debug = YES;
    if (!debug)debug= NO;
    if (_debug != debug){
        NSLog(@"设置调试模式：%@", debug?@"YES":@"NO");
        _debug = debug;
    }
}
+ (void)print:(NSString*)value{
    if (_debug){
        NSLog(@"com.fluttercandies.qweather：%@",value);
    }
}
@end

File: android/src/main/kotlin/com/fluttercandies/flutter_qweather/DebugPrint.java
package com.fluttercandies.flutter_qweather;

/// 调试打印
public class DebugPrint {
    private static boolean _debug = false;


    public static void setDebug(boolean debug) {
        if (_debug != debug){
            System.out.println("com.fluttercandies.qweather: 设置调试模式：" + debug);
            _debug = debug;
        }
    }

    public static void print(String value) {
        if (_debug) System.out.println("com.fluttercandies.qweather: " + value);
    }
}
